src/main/java/main/model/db/imports/ImportHandlers/Cucumber.java

```diff
@@ -58,6 +65,9 @@ private void handleFeature(@NotNull FeatureDto feature) {
         }
 
         private void handleScenario(@NotNull ScenarioDto scenario) {
+            if(scenario.getStart_timestamp() != null){
```

Comment:
can we join together fetching of startTime and finish time and move it into a separate method that takes scenario.getStart_timestamp() as a parameter?

startTime = scenario.getStart_timestamp().before(startTime) ? scenario.getStart_timestamp() : startTime;
finishTime = Date.from(scenario.getStart_timestamp().toInstant().plusMillis(testTime)).after(finishTime)
                            ? Date.from(scenario.getStart_timestamp().toInstant().plusMillis(testTime))
                            : finishTime;